### PR TITLE
init: finish task-5

### PR DESCRIPTION
**Pull request by izzetbek**
Front-end: https://d3hmyqwjwvay0w.cloudfront.net/

1 - File serverless.yml contains configuration for importProductsFile function
3 - The importProductsFile lambda function returns a correct response which can be used to upload a file into the S3 bucket
4 - Frontend application is integrated with importProductsFile lambda
5 - The importFileParser lambda function is implemented and serverless.yml contains configuration for the lambda

Additional (optional) tasks
+1 - async/await is used in lambda functions
+1 - importProductsFile lambda is covered by unit tests (aws-sdk-mock can be used to mock S3 methods - https://www.npmjs.com/package/aws-sdk-mock)
+1 - At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder)

### CODE REVIEW

**Comment by tambovchanin** [comment]
- [x] - File serverless.yml contains configuration for importProductsFile function
- [ ] - The importProductsFile lambda function returns a correct response which can be used to upload a file into the S3 bucket
Error Message

- [x] - Frontend application is integrated with importProductsFile lambda
- [x] - The importFileParser lambda function is implemented and serverless.yml contains configuration for the lambda
- [x] - async/await is used in lambda functions
- [x] - importProductsFile lambda is covered by unit tests (aws-sdk-mock can be used to mock S3 methods - https://www.npmjs.com/package/aws-sdk-mock)
- [ ] - At the end of the stream the lambda function should move the file from the uploaded folder into the parsed folder (move the file means that file should be copied into parsed folder, and then deleted from uploaded folder)

**Total - 5**

 По всей видимости Url API для подписания запроса надо поправить.

**Comment by izzetbek** [comment]
Спасибо, поправлю

**Comment by EugeneCh** [comment]
Frontend upload is failing with CORS, try to set CORS bucket policy for get, put and delete at least

**Comment by EugeneCh** [comment]
I can't check signedurl because lambda
https://gblhr1x62d.execute-api.eu-west-1.amazonaws.com/dev/import
returns "Missing Authentication Token"

**Comment by izzetbek** [comment]
> I can't check signedurl because lambda
> https://gblhr1x62d.execute-api.eu-west-1.amazonaws.com/dev/import
> returns "Missing Authentication Token"

You probably open this link with incorrect method. Have issues now with cors, but error you mentioned is thrown usually when incorrect resource is requested.

File: src/import-service/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  importProductsFile:
+    handler: handler.importProductsFile
+    events:
+      - http:
+          path: import
+          method: get
+          cors: true
+          request:
+            parameters:
+            querystrings:
```
Comment (sleepwalky) [inline src/import-service/serverless.yml]: you're missing 1 level of indentation here

Comment (izzetbek) [reply src/import-service/serverless.yml]: Fixed this, thank you

**Pull request merged**